src/lib.rs

```diff
@@ -529,3 +598,41 @@ impl Display for Error {
 }
 
 impl std::error::Error for Error {}
+
+// When converting CFF2 to TTF, we need to write a version 1.0 MAXP table.
+pub(crate) struct MaxpData {
```

Comment:
should this maybe go in the maxp.rs file? and the visibilities can then just be pub as the struct itself won't be reexported.

(I realize that you always use `pub(crate)` which also works and one could argue which is better, but for now the style in this repository is to use just `pub` if the item is in a private module anyway.)